Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.10",
+	"GoVersion": "go1.12",
```

Comment:
We shouldn't be changing the go version here. 1.12 is still on go1.10

Reply:
good catch. Let me update the vendor again by the correct Go version

Reply:
fixed now